src/lib/event.js

```diff
@@ -1,24 +1,23 @@
-import { promisify as p } from 'util';
 import validator from 'validator';
 import { removeAll as removeAllResponses } from './responses';
 
 const db = require.main.require('./src/database');
 const plugins = require.main.require('./src/plugins');
 const posts = require.main.require('./src/posts');
 
-const sortedSetAdd = p(db.sortedSetAdd);
-const sortedSetRemove = p(db.sortedSetRemove);
-const getSortedSetRangeByScore = p(db.getSortedSetRangeByScore);
-const getSortedSetRange = p(db.getSortedSetRange);
-// const getObjectsFields = p(db.getObjectsFields);
-const setObject = p(db.setObject);
-const getObject = p(db.getObject);
-const getObjects = p(db.getObjects);
-const deleteKey = p(db.delete);
-const exists = p(db.exists);
-const fireHook = p(plugins.fireHook);
-const getCidsByPids = p(posts.getCidsByPids);
-const getCidByPid = p(posts.getCidByPid);
```

Comment:
These could of course be reduced to a single line, but I left it as-is for refactoring later.